packages/connect-node-test/src/crosstest/timeout_on_sleeping_server.spec.ts

```diff
@@ -48,45 +47,47 @@ xdescribe("timeout_on_sleeping_server", function () {
       // We expect this to be DEADLINE_EXCEEDED, however envoy is monitoring the stream timeout
       // and will return an HTTP status code 408 when stream max duration time reached, which
       // cannot be translated to a connect error code, so connect-web client throws an Unknown.
-      expect(
-        err.code === Code.Unknown || err.code === Code.DeadlineExceeded
-      ).toBeTrue();
+      expect(err.code === Code.DeadlineExceeded).toBeTrue();
     }
   }
-  servers.describeTransports((transport) => {
-    it("with promise client", async function () {
-      const client = createPromiseClient(TestService, transport());
-      try {
-        for await (const response of client.streamingOutputCall(
+  servers.describeTransportsExcluding(
+    // TODO(TCN-918)
+    ["@bufbuild/connect-node (gRPC, binary, http2) against grpc-go (h2)"],
+    (transport) => {
+      it("with promise client", async function () {
+        const client = createPromiseClient(TestService, transport());
+        try {
+          for await (const response of client.streamingOutputCall(
+            request,
+            options
+          )) {
+            fail(
+              `expecting no response from sleeping server, got: ${response.toJsonString()}`
+            );
+          }
+          fail("expected to catch an error");
+        } catch (e) {
+          expectError(e);
```

Comment:
You can simplify this now, and remove the function expectError completely:

```suggestion
          expect(e).toBeInstanceOf(ConnectError);
          expect(connectErrorFromReason(e).code).toBe(Code.DeadlineExceeded);
```